index.html

```diff
@@ -6,8 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" href="vendor/styles/normalize.css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
-    <link href='https://fonts.googleapis.com/css?family=Questrial|Quattrocento|Fanwood+Text' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700|Fanwood+Text' rel='stylesheet' type='text/css'>
   </head>
 
   <body>
```

Comment:
It will make things much easier if you maintain consistent indentation in your HTML. Remember to add the linter to help you maintain it.